modules/gdpr_fields/gdpr_fields.install

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/**
+ * @file
+ * Install file for the GDPR Fields module.
+ */
+
+/**
+ * Implements hook_schema().
+ */
+function gdpr_fields_schema() {
+  $schema['gdpr_fields_field_data'] = array(
+    'description' => 'Stores GDPR field data.',
+    'export' => array(
+      'key' => 'name',
+      'key name' => 'Name',
+//      'admin_title' => 'label',
+//      'admin_description' => 'description',
+      'primary key' => 'name',
+      'identifier' => 'field',
+      'default hook' => 'gdpr_fields_default_field_data',
+      'object' => 'GDPRFieldData',
+    ),
+    'fields' => array(
+      'name' => array(
+        'type' => 'varchar',
+        'length' => 255,
+        'description' => 'Machine name for field.',
+      ),
+      'plugin_type' => array(
+        'type' => 'varchar',
+        'length' => 32,
+        'description' => 'Plugin type of the field.',
+      ),
+      'entity_type' => array(
+        'type' => 'varchar',
+        'length' => 32,
+        'description' => 'Entity type of field.',
+      ),
+      'entity_bundle' => array(
+        'type' => 'varchar',
+        'length' => 32,
+        'description' => 'Entity bundle of field.',
+      ),
+      'field_name' => array(
```

Comment:
Consider changing to word 'property' as I understand this is more about properties than fields.